freebsd-kvm/buildkite-worker/kvm_machine.pkr.hcl.template

```diff
@@ -46,6 +46,16 @@ source "qemu" "freebsd12" {
 build {
     sources = ["source.qemu.freebsd12"]
 
+    provisioner "file" {
+        source = "../../secrets"
+        destination = "/tmp/secrets"
+    }
+
+    provisioner "file" {
+        source = "../../hooks"
+        destination = "/tmp/hooks"
+    }
```

Comment:
FWIW based on my read of the Packer documentation, this should be equivalent but I'm not entirely sure.
```suggestion
    provisioner "file" {
        sources = [
            "../../secrets",
            "../../hooks",
        ]
        destination = "/tmp/"
    }
```